Select by ID/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Phrase Sayer</title>
+  </head>
+  <body>
+    <p><label for="input-phrase">Type a phrase</label></p>
+    <p><input type="text" id="input-phrase" /></p>
+    <p><button id="btn-phrase">Say Phrase</button></p>
```

Comment:
We should not wrap these elements in a `p` tag. This structure violates HTML semantic standards.

Reply:
Thanks, @lycoangelo, This is now updated.